pkg/pillar/cmd/volumemgr/initialvolumestatus.go

```diff
@@ -296,23 +232,6 @@ func gcResetPersistObjectLastUse(ctx *volumemgrContext) {
 	}
 }
 
-// gc timer just started, reset the LastUse timestamp
-func gcResetObjectsLastUse(ctx *volumemgrContext, dirName string) {
-
-	log.Debugf("gcResetObjectsLastUse()")
-
-	pub := ctx.publication(types.OldVolumeStatus{}, types.UnknownObj)
```

Comment:
I think you need this for VolumeStatus as well, including calling it when the Old function is called.

Reply:
Added.